class-wc-facebookcommerce.php

```diff
@@ -383,8 +384,8 @@ public function add_wordpress_integration() {
 		 * @param string $log_id optional log id to segment the files by, defaults to plugin id
 		 */
 		public function log( $message, $log_id = null ) {
-			// bail if logging isn't enabled
-			if ( ! $this->get_integration() || ! $this->get_integration()->is_debug_mode_enabled() ) {
+			// bail if logging isn't enabled, always enabled if is not connected.
```

Comment:
Confusing logic and comment! Does something like this make it clearer?

`Bail if site is connected and user has disabled logging`

Probably worth explicitly documenting that we force logging if site is disconnected so it's easier to support/diagnose. So maybe…

```php
// Bail if site is connected and user has disabled logging.
// If site is disconnected, force-enable logging so merchant can diagnose connection issues.

Reply:
thank you for the suggestion, updated in ff9b59f